### PR TITLE
Make seed "global" for all methods and place it more prominent

### PR DESCRIPTION
**Pull request by koenderks**
Feedback from in-class testing @ nyenrode

### CODE REVIEW

File: inst/qml/auditSelection.qml [hunk 1]
```diff
@@ -84,14 +84,27 @@ Form
...
+		IntegerField
+		{
+			id: 								seed
+			text: 								qsTr("Seed")
+			name: 								"seed"
+			defaultValue: 						1
+			min: 								1
+			max: 								99999
+			enabled:							randomize.checked | method.value != "interval"
+		}
```
Comment (boutinb) [inline inst/qml/auditSelection.qml:107]: The '|' is a bit operator, use '||' in place

**Review by boutinb — CHANGES_REQUESTED** [review]

File: inst/qml/auditSelection.qml [hunk 2]
```diff
@@ -84,14 +84,27 @@ Form
...
+		IntegerField
+		{
+			id:									nobs
+			text: 								qsTr("Sample size")
+			name: 								"n"
+			defaultValue: 						0
+			min: 								0
+		}
+
+		IntegerField
```
Comment (vandenman) [inline inst/qml/auditSelection.qml:98]: While this works, there is also the control `SetSeed{}`. You don't want to use that?

File: inst/qml/auditClassicalWorkflow.qml [hunk 3]
```diff
@@ -774,8 +769,8 @@ Form
 			Button
 			{
 				enabled:							((materiality_rel.checked ?
-														  materiality_rel_val.value != "0" && id.count > 0 :
-														  materiality_abs_val.value 		!= "0" && id.count > 0 && values.count > 0) |
+														materiality_rel_val.value != "0" && id.count > 0 :
+														materiality_abs_val.value != "0" && id.count > 0 && values.count > 0) |
```
Comment (vandenman) [inline inst/qml/auditClassicalWorkflow.qml:773]:
```suggestion
														materiality_abs_val.value != "0" && id.count > 0 && values.count > 0) ||
```

File: inst/qml/auditBayesianWorkflow.qml [hunk 4]
```diff
@@ -906,8 +901,8 @@ Form
 			Button
 			{
 				enabled:							((materiality_rel.checked ?
-														  materiality_rel_val.value != "0" && id.count > 0 :
-														  materiality_abs_val.value 		!= "0" && id.count > 0 && values.count > 0) |
+														materiality_rel_val.value != "0" && id.count > 0 :
+														materiality_abs_val.value != "0" && id.count > 0 && values.count > 0) |
```
Comment (vandenman) [inline inst/qml/auditBayesianWorkflow.qml:905]:
```suggestion
														materiality_abs_val.value != "0" && id.count > 0 && values.count > 0) ||
```

**Review by vandenman — CHANGES_REQUESTED** [review]

File: inst/qml/auditSelection.qml (hunk 2 above)
Comment (koenderks) [reply inst/qml/auditSelection.qml:98]: I saw it comes with a groupbox and a checkbox, but that is not ideal here

**Review by boutinb — APPROVED** [review]

**Review by vandenman — APPROVED** [review]

**Pull request merged**